### PR TITLE
PXB-3222 Redo the option topics for xtrabackup, xbstream, xbcloud, an…

### PR DESCRIPTION
**Pull request by patrickbirch**
…d xbcrypt (8.0)

	modified:   docs/xbcloud-binary-overview.md
	new file:   docs/xbcloud-options.md
	modified:   docs/xbcloud-swift.md
	modified:   docs/xbcrypt-binary-overview.md
	new file:   docs/xbcrypt-options.md
	modified:   docs/xbstream-binary-overview.md
	new file:   docs/xbstream-options.md
	modified:   docs/xtrabackup-option-reference.md
	modified:   mkdocs-base.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by satya-bodapati — CHANGES_REQUESTED** [review]
I see some text in options misaligned
xtrbackup option reference can be renamed as xtrbackup commandline options (similar to the other two binaries)
braces/brackets at end of each option can be avoided

File: docs/xbcloud-swift.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 ## Create a full backup with Swift
 
+For details on the command-line options,  see the [xbcloud command-line options]
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-swift.md:5]:
```suggestion
For details on the command-line options, see the [xbcloud command-line options].
```

File: docs/xbcloud-swift.md [hunk 2]
```diff
@@ -2,9 +2,11 @@
...
+
 The following example shows how to make a full backup and upload it to Swift.
 
-```{.bash data-prompt="$"}
+```{.bash
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-swift.md:9]:
```suggestion
```{.bash data-prompt="$"}
```

File: docs/xbcloud-swift.md [hunk 3]
```diff
@@ -16,51 +18,40 @@ full_backup
...
-
 * OS_STORAGE_URL
 
-
 * OS_CACERT
 
 ## Restore with Swift
 
-```{.bash data-prompt="$"}
+```{.bash
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-swift.md:48]:
```suggestion
```{.bash data-prompt="$"}
```

File: docs/xbcloud-swift.md [hunk 4]
```diff
@@ -16,51 +18,40 @@ full_backup
...
 $ xbcloud get [options] <name> [<list-of-files>] | xbstream -x
 ```
 
 The following example shows how to fetch and restore the backup from Swift:
 
-```{.bash data-prompt="$"}
+```{.bash
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-swift.md:54]:
```suggestion
```{.bash data-prompt="$"}
```

File: docs/xbcrypt-binary-overview.md [hunk 5]
```diff
@@ -5,45 +5,6 @@ introduced to *Percona XtraBackup*.
 
 **Percona XtraBackup** 8.0.28-20 implements the XBCRYPT_ENCRYPTION_KEY environment variable. The variable is only used in place of the `--encrypt_key=name` option. You can use the environment variable or command line option. If you use both, the command line option takes precedence over the value specified in the environment variable.
 
-This utility has been modeled after The [xbstream binary](xbstream-binary-overview.md) to perform
-encryption and decryption outside of *Percona XtraBackup*. `xbcrypt` has
-following command line options:
+This utility has been modeled after The [xbstream binary](xbstream-binary-overview.md) to perform encryption and decryption outside of *Percona XtraBackup*. `
```
Comment (alina-derkach-oaza) [inline docs/xbcrypt-binary-overview.md:8]:
```suggestion
This utility has been modeled after the [xbstream binary](xbstream-binary-overview.md) to perform encryption and decryption outside of *Percona XtraBackup*.
```

File: docs/xbcrypt-options.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
+# xbcrypt command-line options
+
+xbcrypt` has following command line options:
```
Comment (alina-derkach-oaza) [inline docs/xbcrypt-options.md]:
```suggestion
`xbcrypt` has the following command line options:
```

**Comment by satya-bodapati** [comment]
galera-info[¶](https://percona-xtrabackup-docs-8-0-pr-233.onrender.com/xtrabackup-option-reference.html#galera-info)

Usage: --galera-info(

This option creates the xtrabackup_galera_info file which contains the local node state at the time of the backup. Option should be used when performing the backup of Percona XtraDB Cluster. It has no effect when backup locks are used to create the backup.
generate-new-master-key[¶](https://percona-xtrabackup-docs-8-0-pr-233.onrender.com/xtrabackup-option-reference.html#generate-new-master-key)

Usage: --generate-new-master-key(


Two extra braces in the usage.

Rest all looks good Patrick! Thank you for fixing this

File: docs/xtrabackup-option-reference.md [hunk 7]
```diff
@@ -1,319 +1,328 @@
...
 backup. It is very important for incremental backups.
 
+### backup
+
+Usage: `--backup`
 
-### backup()
-Make a backup and place it in `--target-dir`. See
-Creating a backup.
+Make a backup and place it in `--target-dir`. See [Creating a full backup](create-full-backup.md).
```
Comment (alina-derkach-oaza) [inline docs/xtrabackup-option-reference.md]:
```suggestion
Make a backup and place it in `--target-dir`. See [Create a full backup](create-full-backup.md).
```

File: docs/xbcrypt-options.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
+# xbcrypt command-line options
```
Comment (alina-derkach-oaza) [inline docs/xbcrypt-options.md]:
```suggestion
# The xbcrypt command-line options
```

File: docs/xbstream-options.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
+# xbstream command-line options
```
Comment (alina-derkach-oaza) [inline docs/xbstream-options.md]:
```suggestion
# The xbstream command-line options
```

File: docs/xbcloud-options.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
+# xbcloud command-line options
```
Comment (alina-derkach-oaza) [inline docs/xbcloud-options.md]:
```suggestion
# The xbcloud command-line options
```

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**